docs/source/quickstart.rst

```diff
@@ -59,6 +59,9 @@ You can then run MLflow's Tracking UI:
 
 and view it at `<http://localhost:5000>`_. 
 
+.. note::
```

Comment:
Thanks for addressing this @dennyglee. I was reading up about this, the recommendation was to use `localhost` to be ipv6 compliant.

I was going to try this out `--worker-class 'eventlet'` as recommended in these  [lecture notes](http://www.cs.fsu.edu/~carnahan/cis4930sp17/Lecture15_cis4930.pdf).

There was another recommendation to [increase `timeouts`, used `graceful_timeout`, and `keep_alive`](http://docs.gunicorn.org/en/stable/settings.html#timeout).

Reply:
Sure, I can give that a try - silly question, since I'm running this via `mlflow ui`, where should I make the change for the `gunicorn` call?

Reply:
here : https://github.com/databricks/mlflow/blob/master/mlflow/server/__init__.py#L47

Thanks!

Reply:
Hmm! Is there a way to reproduce it `mlflow server`. Lemme try as well..

Reply:
Did you guys end up investigating this? Maybe we should just merge this change and then try changing the worker type later.

Reply:
That's a good idea. Lemme incorporate @stbof 's comment and merge this in.